cmd/dcos/main.go

```diff
@@ -23,13 +31,21 @@ func run(ctx api.Context) error {
 }
 
 func errorMessage(err error) string {
-	if httpErr, ok := err.(*httpclient.HTTPError); ok {
-		switch httpErr.Response.StatusCode {
+	switch e := err.(type) {
+	case *httpclient.HTTPError:
+		switch e.Response.StatusCode {
 		case 401:
 			return "authentication failed, please run `dcos auth login`"
 		case 403:
 			return "you are not authorized to perform this operation"
 		}
+	case *url.Error:
+		switch e.Err.(type) {
+		case x509.CertificateInvalidError:
```

Comment:
```suggestion
		case x509.CertificateInvalidError, x509.UnknownAuthorityError:
```
https://github.com/golang/go/wiki/Switch#multiple-cases